packages/hidp_wagtail/hidp_wagtail/static/css/wagtail_hidp.css

```diff
@@ -33,7 +33,8 @@ h2 {
   font-size: 1.25rem;
 }
 
-button {
+button,
+form[action="/logout/"] button.primary {
```

Comment:
How can we rely on the action being exactly `"/logout/"`? Doesn't that depend on how/where you configured the urlpatterns?

Reply:
Indeed, it's not the best way to select these forms. I changed it to use $= selector instead (ends with).

Reply:
> Since the styles and the templates are part of the same theme-pack, maybe we can allow ourselves to use id's? I.e. `form#logout button.primary {`

this ^